base/task.jl

```diff
@@ -4,6 +4,49 @@
 
 show(io::IO, t::Task) = print(io, "Task ($(t.state)) @0x$(hex(convert(UInt, pointer_from_objref(t)), WORD_SIZE>>2))")
 
+type CollectedException <: Exception
+    pid::Int
+    ex
+    bt
```

Comment:
We might want to have a standard way to encapsulate an Exception and a backtrace, without necessarily including a pid.